gamestonk_terminal/cryptocurrency/cryptocurrency_helpers.py

```diff
@@ -30,11 +29,34 @@
     show_available_pairs_for_given_symbol,
     plot_candles,
 )
+
+from gamestonk_terminal.cryptocurrency.due_diligence import coinbase_model
 import gamestonk_terminal.config_terminal as cfg
 from gamestonk_terminal.feature_flags import USE_ION as ion
 from gamestonk_terminal import feature_flags as gtff
 
 
+def _load_coin_map(file_name: str) -> pd.DataFrame:
+    if file_name.split(".")[1] != "json":
+        raise TypeError("Please load json file")
+
+    path = os.path.abspath(__file__ + "/../")
```

Comment:
Instead of using "/" on the str, you can use `os.path.join()`

Reply:
Thanks! I refactored it :)